crates/bevy_render/src/color.rs

```diff
@@ -37,6 +37,21 @@ impl Color {
     pub const fn rgba(r: f32, g: f32, b: f32, a: f32) -> Color {
```

Comment:
I think we could do with some doc comments on `Color` and  its associated functions about being normalised values.

Reply:
Yeah i think that probably makes sense, both on the Color struct and on functions taking floats (rgba/rgb). Probably should be a separate PR though. Feel free to create one (or an issue to track it).